testground/benchmark/benchmark/sendtx.py

```diff
@@ -77,12 +77,12 @@ def sendtx(w3: web3.Web3, acct: Account, tx_amount: int):
     )
 
 
-def generate_load(cli, num_accounts, num_txs, **kwargs):
+def generate_load(cli, num_accounts, num_txs, num_max_workers, **kwargs):
     w3 = web3.Web3(web3.providers.HTTPProvider("http://localhost:8545"))
     assert w3.eth.chain_id == 777
     genesis_account = export_eth_account(cli, "account", **kwargs)
     accounts = fund_test_accounts(w3, genesis_account, num_accounts)
-    with ThreadPoolExecutor(max_workers=num_accounts) as executor:
+    with ThreadPoolExecutor(max_workers=num_max_workers) as executor:
         futs = (executor.submit(sendtx, w3, acct, num_txs) for acct in accounts)
```

Comment:
task is based on num_max_workers, not based on account

Reply:
aren't we only `executor.submit` one task per account?

Reply:
I see, it's only useful when less than num_accounts to avoid OOM

Reply:
yes, but the only reason to use too many num_accounts is only to improve concurrency level